modules/openy_gc_auth/modules/openy_gc_auth_example/openy_gc_auth_example.libraries.yml

```diff
@@ -0,0 +1,5 @@
+openy_gc_auth_example:
+  js:
```

Comment:
We should always have version for all libraries - see https://www.drupal.org/node/3281083

Reply:
Thanks @podarok.